roles/robots_txt/templates/var/www/robots.txt

```diff
@@ -0,0 +1,20 @@
+{% if archive_robots_txt.sitemap_name is defined %}
+Sitemap: https://{{ frontend_domain }}/{{ robots_txt.sitemap_name }}
+{% endif %}
+{% for user_agent in archive_robots_txt.user_agents %}
+User-agent: {{ user_agent.name }}
+{% if user_agent.crawl_delay|default(None) %}
+Crawl-delay: {{ user_agent.crawl_delay }}
+{% endif %}
+{% if archive_robots_txt.disallow_list is defined %}
+{% for disallow in archive_robots_txt.disallow_list %}
+Disallow: {{ disallow }}
+{% endfor %}
+{% endif %}
+{% if archive_robots_txt.allow_list is defined %}
+{% for allow in archive_robots_txt.allow_list %}
+Allow: {{ allow }}
```

Comment:
I wonder why the webview robots.txt doesn't have an Allow section.

Reply:
Seems like it might be a good idea to collapse this into a single temple file. But It's ok as is as well.

Reply:
Yeah, I thought about making it a single template, but my ansible-fu is weak: I didn't know how to change the variable values for different calls to template. Checking stackoverflow, this seems to be a common ansible complaint/weakness.

Reply:
I jus spent 30 min poking at what should have been a sane solution (using include_var) but ansible is not sane.

Reply:
one of us, one of us